e2e/visual/no-theme.spec.js

```diff
@@ -61,7 +61,7 @@ test('no-theme - editor', async ({ page, makeAxeBuilder }) => {
   });
 
   // when
-  await page.goto('/carbon');
+  await page.goto('/carbon/');
```

Comment:
Something changed on Vite's HTTP server that they use for the preview command, it doesn't match the carbon folder without the slash...